model/connect/PDOConnector.php

```diff
@@ -159,6 +159,22 @@ public function connect($parameters, $selectDB = false) {
 		$options = array();
 		if($parameters['driver'] == 'mysql') {
 			$options[PDO::MYSQL_ATTR_INIT_COMMAND] = 'SET NAMES ' . $charset . ' COLLATE ' . $connCollation;
+
+			// Set SSL options from mysite/_config.php if they exist
+			if(
+				array_key_exists('ssl_key', $parameters) &&
+				array_key_exists('ssl_cert', $parameters) &&
+				array_key_exists('ssl_ca', $parameters) &&
+				array_key_exists('ssl_cipher', $parameters)) {
+
+					$options[PDO::MYSQL_ATTR_SSL_KEY] = $parameters['ssl_key'];
+					$options[PDO::MYSQL_ATTR_SSL_CERT] =$parameters['ssl_cert'];
+					$options[PDO::MYSQL_ATTR_SSL_CA] = $parameters['ssl_ca'];
+					$options[PDO::MYSQL_ATTR_SSL_CIPHER] = $parameters['ssl_cipher'];
```

Comment:
Lets make this a config rather than hard coding it.

```php
private static $ssl_cipher_default = 'DHE-RSA-AES256-SHA';
// ...
$options[PDO::MYSQL_ATTR_SSL_CIPHER] = $parameters['ssl_cipher'] ?: $this->config()->ssl_cipher_default;
```